tools/vmd/BUILD.bazel

```diff
@@ -22,19 +22,19 @@ sh_binary(
     name = "ios_vm_test_runner",
     srcs = ["ios_vm_test_runner.sh"],
     data = [
-        ## We want to call the original runner under vm_run
+        ## We want to call the original runner under vmd
         ":vmd",
         "@xctestrunner//:ios_test_runner",
     ],
     visibility = ["//visibility:public"],
 )
 
-# Wrap everything into a CLI
+# Wrap everything into a CLI: this has to unfortunately replicate deps
```

Comment:
yeah, unfortunately `sh_binary` isn't as nice as most of the others re: bundling 😞